vision_agent_tools/models/internlm_xcomposer2.py

```diff
@@ -76,7 +76,8 @@ def __call__(
         image: Image.Image | None = None,
         video: VideoNumpy | None = None,
         frames: Frames = MAX_NUMBER_OF_FRAMES,
-    ) -> str:
+        chunk_size: int = 1,
```

Comment:
let's call it `n_chunks` instead, `chunk size` could be confusing.

Reply:
Done! I changed the name